trlx/trainer/accelerate_base_trainer.py

```diff
@@ -78,19 +78,32 @@ def __init__(self, config, **kwargs):
             dist_config = get_distributed_config(self.accelerator)
             config_dict["distributed"] = dist_config
             init_trackers_kwargs = {}
-            if "wandb" in config.train.trackers:
+            # HACK: Tensorboard doesn't like nested dict as hyperparams
+            config_dict_flat = {a:b for (k,v) in config_dict.items() for (a,b) in v.items() if not isinstance(b, dict)}
```

Comment:
You can use trlx.utils.modeling.flatten_dict here

Reply:
thanks for the suggestion! I replaced the dict comprehension with a call to flatten_dict(), since tensorboard also doesn't like lists I added a couple lines to split the optimizer betas